src/main/java/website/frontrow/game/Game.java

```diff
@@ -61,9 +60,29 @@ public void tick()
             {
                 levelWon();
             }
+            
+            if(playersDead())
+            {
+            	levelLost();
+            }
         }
         updateObservers();
     }
+    
+    /**
+     * This is used by just a few tests that didn't work the way they
+     * should've worked, because they used a mocked level.
+     * Because the level has no enemies, the game would stop.
+     * In order for the tests to be accurate, I made a new tick
+     * function that only checks whether or not the players are dead.
```

Comment:
Dat snap ik, maar ik heb dat nu al een tijdje zitten proberen en dat lukt nu niet. Laten we het maar zo houden voor deze week, dan zit de lives en game over feature in ieder geval al in het spel en dan fixen we dit de volgende keer wel.